study.md

```diff
@@ -158,5 +180,7 @@ What is a response? what does it contain? What are some common status codes in a
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+<!-- curl https://reddit.com/
+
+is that it? I thought it would be something like curl GET /https://www.reddit.com/ or something like that but it seems like just curl and the url does will get you the content of the url and -->
```

Comment:
yea there is a few different ways to do it. You might have to add the www in front of reddit though.